src/js/jsx/tools/SuperselectOverlay.jsx

```diff
@@ -388,6 +387,58 @@ define(function (require, exports, module) {
             this._currentMouseDown = true;
         },
 
+        updateMouseOverHighlights: function () {
+            var marquee = this.state.marqueeEnabled,
+                scale = this._scale,
+                uiStore = this.getFlux().store("ui"),
+                mouseX = this._currentMouseX,
+                mouseY = this._currentMouseY,
+                canvasMouse = uiStore.transformWindowToCanvas(mouseX, mouseY),
+                highlightFound = false;
+
+            // Yuck, we gotta traverse the list backwards, and D3 doesn't offer reverse iteration
+            _.forEachRight(d3.selectAll(".layer-bounds")[0], function (element) {
+                var layer = d3.select(element),
+                    layerLeft = parseInt(layer.attr("x")),
```

Comment:
Don't use `window.parseInt`. It's weird. You have to supply a radix, and if it fails you get NaN which is strangely hard to test for. Instead use `parseNumber` from `js/util/math`, which returns null when it fails to parse.